backend/server/routers/planner.py

```diff
@@ -91,11 +88,24 @@ def validate_term_planner(plannerData: PlannerData):
         }
     }
 )
-def autoplanning(courseCodes: List, userData: UserData, programTime: ProgramTime) -> Dict:
-    user = User(dict(userData))
-
+def autoplanning(courseCodes: List, plannerData: PlannerData, programTime: ProgramTime) -> Dict:
```

Comment:
also, please add tests :)

Reply:
also i just noticed - shouldnt this be a POST? I know we are not actually changing anything, but we probably want to let the FE use the body to make a request.